components/org.wso2.carbon.identity.application.authenticator.oidc/pom.xml

```diff
@@ -156,7 +162,13 @@
                             version="${identity.framework.package.import.version.range}",
                             org.wso2.carbon.ui; version="${carbon.kernel.package.import.version.range}",
                             com.nimbusds.jose.*; version="${nimbusds.osgi.version.range}",
-                            com.nimbusds.jwt; version="${nimbusds.osgi.version.range}"
+                            com.nimbusds.jwt; version="${nimbusds.osgi.version.range}",
+                            com.nimbusds.oauth2.sdk.id.*;
```

Comment:
Any reason why we haven't added the OSGi version range?

Reply:
dependecy is removed in 789a83b